docs/developerguide.rst

```diff
@@ -56,3 +56,135 @@ We are using Prettier to format our code. In order to use it before a Pull Reque
    .. code-block:: console
 
       $ prettier --write **/*.js
+
+CERN
```

Comment:
What do you think of `CERN integrations locally`? Just CERN may sound a bit too broad.